docs/guides/hardhat-walkthrough/_category_.yml

```diff
@@ -0,0 +1,3 @@
+label: '✏️ Hardhat Walkthrough'
+collapsed: true
+position: 1
```

Comment:
maybe this should not be 1 as network is currently at position one.

Reply:
Where should I put it? After browser extension maybe?

Reply:
I put it at position 3